cmd/build/command.go

```diff
@@ -80,21 +83,29 @@ func Build(ctx context.Context) *cobra.Command {
 	return cmd
 }
 
-func (bc *Command) getManifestAndBuildVersion(options *types.BuildOptions) (*model.Manifest, bool) {
-	manifest, errManifest := bc.GetManifest(options.File)
-	if errManifest != nil {
-		oktetoLog.Debug("error getting manifest v2 from file %s: %v. Fallback to build v1", options.File, errManifest)
+func (bc *Command) getBuilder(options *types.BuildOptions) (Builder, error) {
+	var builder Builder
+
+	manifest, err := bc.GetManifest(options.File)
+	if err != nil {
+		if errors.Is(err, oktetoErrors.ErrInvalidManifest) {
+			return nil, err
+		}
+
+		oktetoLog.Warning("error getting manifest v2 from file %s: %v. Fallback to build v1", options.File, err)
```

Comment:
Will this be seen by the user? If so, I think that `Warning` may be a bit too high and `Info` may be a better choice. Also, I'd state `The manifest %s is not v2 compatible, falling back to building as a v1 manifest: %v`